mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/TaskExecutor.java

```diff
@@ -116,6 +117,8 @@ public class TaskExecutor extends RpcEndpoint implements TaskExecutorGateway {
 
     private final TaskFactory taskFactory;
 
+    private ExecuteStageRequest currentRequest;
```

Comment:
nit: Move this below `RuntimeTask currentTask;` This way, all the state about the current runtime task is captured clearly.